terraform/node_module_removed.go

```diff
@@ -39,12 +39,12 @@ func (n *NodeModuleRemoved) EvalTree() EvalNode {
 	}
 }
 
-func (n *NodeModuleRemoved) ReferenceOutside() (selfPath, referencePath addrs.ModuleInstance) {
+func (n *NodeModuleRemoved) ReferenceOutside() (selfPath, referencePath addrs.Module) {
```

Comment:
Here we're changing from referring to instances to referring to modules; this is because we want to talk about modules _as a whole_ because one module could expand to many instances. You'll see similar changes further down.